test/validate_v1_v2_equality.ts

```diff
@@ -248,4 +259,108 @@ describe('Printings v1/v2 equality', () => {
   it('stripped_printed_text matches', () => {
     validate('stripped_text', 'stripped_printed_text');
   });
-});
\ No newline at end of file
+});
+
+describe('MWLs v1/v2', () => {
+  const standardRestrictions = getRestrictionsV2Json()['standard'];
+  const mwls = getMwlJson();
+  const pairs = mwls.map(mwl => [mwl, standardRestrictions.find(r => r.name == mwl.name)]);
+
+  const printings = getPrintingsV2Json ();
+  const printingsById = new Map<string, any>();
+  printings.forEach(p => {
+    printingsById.set(p.id, p);
+  });
+  const printingsByCardId = new Map<string, any>();
+  getCardsV2Json ().forEach(c => {
+    printingsByCardId.set(c.id, []);
+  });
+  printings.forEach(p => {
+    printingsByCardId.get(p.card_id).push(p);
+  });
+
+  function v1ToBanned(mwl) {
+    const arr: [string, Record<string, number>][] = Object.entries(mwl.cards);
+    return arr.filter(card => card[1].deck_limit != undefined && card[1].deck_limit == 0)
+              .map(card => card[0]).sort();
+  }
+  function v1ToRestricted(mwl) {
+    const arr: [string, Record<string, number>][] = Object.entries(mwl.cards);
+    return arr.filter(card => card[1].is_restricted)
+              .map(card => card[0]).sort();
+  }
+  function v1ToUniversalFactionCost(mwl) {
+    const arr: [string, Record<string, number>][] = Object.entries(mwl.cards);
+    const obj = {};
+    arr.filter(card => card[1].universal_faction_cost).sort().forEach(function(card) {
+      if (!obj[card[1].universal_faction_cost]) {
+        obj[card[1].universal_faction_cost] = [];
+      }
+      obj[card[1].universal_faction_cost].push(card[0]);
+    });
+    return obj;
+  }
+  function v1ToGlobalPenalty(mwl) {
+    const arr: [string, Record<string, number>][] = Object.entries(mwl.cards);
+    const obj = {};
+    arr.filter(card => card[1].global_penalty).sort().forEach(function(card) {
+      if (!obj[card[1].global_penalty]) {
+        obj[card[1].global_penalty] = [];
+      }
+      obj[card[1].global_penalty].push(card[0]);
+    });
+    return obj;
+  }
+
+  function cardsToPrintings(cards) {
+    if (!cards)
```

Comment:
style thing for me, but i dislike conditionals with no braces.  this isn't incorrect, obviously.